README.rst

```diff
@@ -17,7 +15,7 @@ Enabling
 
 Add extension in ``conf.py``::
 
-    extensions = [ 'sphinxcontrib.verilogdomain' ]
+    extensions = [ 'sphinx-verilog-domain' ]
```

Comment:
Why did you change the name of the package?

Reply:
I change the directory structure to be consistent with other Sphinx extensions like:
- sphinxcontrib-verilog-diagrams
- markdown-code-symlinks

The new package name is a result of that change. I also believe that the same package and extension name simplifies the setup process for the people unfamiliar with the Sphinx. That's why I used `sphinx-verilog-domain` instead of `sphinx_verilog_domain` as an extension name.